perses/rjmc/geometry.py

```diff
@@ -628,8 +658,12 @@ def _logp_propose(self, top_proposal, old_positions, beta, new_positions=None, d
         _logger.info(f"sum of energies: {atoms_with_positions_reduced_potential + reduced_potential_energy}")
         _logger.info(f"magnitude of difference in the energies: {abs(final_context_reduced_potential - atoms_with_positions_reduced_potential - reduced_potential_energy)}")
 
-        energy_mismatch_ratio = (atoms_with_positions_reduced_potential + reduced_potential_energy) / (final_context_reduced_potential)
-        assert (energy_mismatch_ratio < ENERGY_MISMATCH_RATIO_THRESHOLD + 1) and (energy_mismatch_ratio > 1 - ENERGY_MISMATCH_RATIO_THRESHOLD)  , f"The ratio of the calculated final energy to the true final energy is {energy_mismatch_ratio}"
+        #the below assertion is no longer applicable since biasing forces are unaccounted for in the atoms_with_positions_system/context
```

Comment:
since we are allowing for the use of biasing torsions and (potentially, in the future) angles, we are no longer able to validate energy bookkeeping between the nonalchemical and alchemical endstates (i.e. the atoms_with_positions system energy plus the added valence energy from unique new/old terms should be equal to the alchemical endstates.); it was anticipated that this bookkeeping method would not be kept, and was only used as a sanity check in the past to check for energy terms that were being forgotten in the hybrid system